serving/samples/telemetry-go/README.md

```diff
@@ -99,31 +105,51 @@ kubectl get svc knative-ingressgateway -n istio-system --watch
   NAME                     TYPE           CLUSTER-IP     EXTERNAL-IP      PORT(S)                                      AGE
   knative-ingressgateway   LoadBalancer   10.23.247.74   35.203.155.229   80:32380/TCP,443:32390/TCP,32400:32400/TCP   2d
   ```
+  CTRL+C to end watch.
 
-2. When the service is ready, export the ingress hostname and IP as environment variables:
+  Check the status of your route:
+  ```
+  kubectl get route -o yaml
+  ```
+  When the route is ready, you'll see the following fields reported as:
+  ```YAML
+  status:
+    conditions:
+      ...
+      status: "True"
+      type: Ready
+      domain: telemetrysample-route.default.example.com
+      ```
```

Comment:
You need to unindent the ```; right now it thinks the ``` is quoted, and the rest of the text is blockquoted wrong.